jobs/yolov5-web-service/app.py

```diff
@@ -0,0 +1,63 @@
+import io
+import os
+import shutil
+import json
+from PIL import Image
+
+import torch
+import flask
+from flask import Flask, jsonify, url_for, render_template, request, redirect
+
+app = Flask(__name__)
+
+RESULT_FOLDER = os.path.join('static')
+app.config['RESULT_FOLDER'] = RESULT_FOLDER
+
+model_yolov5s = torch.hub.load('ultralytics/yolov5', 'yolov5s', pretrained=True).autoshape() 
+model_yolov5s.eval()
+
+model_yolov5m = torch.hub.load('ultralytics/yolov5', 'yolov5m', pretrained=True).autoshape() 
+model_yolov5m.eval()
+
+# yolov5s
+def get_prediction_yolov5s(img_bytes):
+    img = Image.open(io.BytesIO(img_bytes))
+    imgs = [img]  # batched list of images
+    # Inference
+    results = model_yolov5s(imgs, size=640) 
+    return results
+
+# yolov5m
+def get_prediction_yolov5m(img_bytes):
+    img = Image.open(io.BytesIO(img_bytes))
+    imgs = [img]  # batched list of images
+    # Inference
+    results = model_yolov5m(imgs, size=640) 
+    return results
+
+@app.route('/', methods=['GET', 'POST'])
```

Comment:
You should separate routes `GET` and `POST` in 2 separates method in your code. By doing this you can then avoid doing a `request.method == 'POST'` in the body of the `predict` function